### PR TITLE
Feature/onboard flow

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

**Comment by RV-Adrian** [comment]
@Rise-Vision/apps 
Please review PR for https://trello.com/c/sVwBD0v1/2392-ux-onboarding-progress-bar-step-by-step-prototype-3

Recorded a breakdown of code here https://drive.google.com/a/risevision.com/file/d/0B8XS4J7-uR-eRXFqeWRrUktkVUU/view?usp=sharing

Thanks!

File: web/partials/editor/workspace.html [hunk 1]
```diff
@@ -1,8 +1,61 @@
...
+    </div>
+</div>
+
+
 <style type="text/css">
 	.navbar .container {width: 100%;}
 </style>
 
-<div id="workspace" class="workspace" ng-class="{'hide-sidebar' : artboardFactory.hideSidebar(), 'active-banner-mode' : isSubcompanySelected() || isTestCompanySelected(), 'active-banner-mode-double' : isSubcompanySelected() && isTestCompanySelected()}">
+<div id="workspace" class="workspace active-onboard-mode" ng-class="{'hide-sidebar' : artboardFactory.hideSidebar(), 'active-banner-mode' : isSubcompanySelected() || isTestCompanySelected(), 'active-banner-mode-double' : isSubcompanySelected() && isTestCompanySelected()}">
```
Comment (alex-deaconu) [inline web/partials/editor/workspace.html]: @RV-Adrian to confirm, we need to be removing the `active-onboard-mode` class when onboarding is complete? thanks.

Comment (alex-deaconu) [inline web/partials/editor/workspace.html]: Disregard, video explains. Thanks.

**Comment by alex-deaconu** [comment]
PR no longer needed as the feature was released. Closing.

**Pull request closed without merging**